sdk/communication/Azure.Communication.JobRouter/src/Models/CreateExceptionPolicyOptions.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.Communication.JobRouter
+{
+    /// <summary>
+    /// Options for creating exception policy.
+    /// </summary>
+    public class CreateExceptionPolicyOptions
+    {
+        /// <summary>
+        /// Public constructor.
+        /// </summary>
+        /// <param name="exceptionPolicyId"> Id of the policy. </param>
+        /// <param name="exceptionRules"> A dictionary collection of exception rules on the exception policy. Key is the Id of each exception rule. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="exceptionPolicyId"/> is null. </exception>
+        /// <exception cref="ArgumentNullException"> <paramref name="exceptionRules"/> is null. </exception>
+        public CreateExceptionPolicyOptions(string exceptionPolicyId, IDictionary<string, ExceptionRule> exceptionRules)
+        {
+            Argument.AssertNotNullOrWhiteSpace(exceptionPolicyId, nameof(exceptionPolicyId));
+            Argument.AssertNotNullOrEmpty(exceptionRules, nameof(exceptionRules));
+
+            ExceptionPolicyId = exceptionPolicyId;
+            ExceptionRules = exceptionRules;
+        }
+
+        /// <summary>
+        /// The Id of this policy.
+        /// </summary>
+        public string ExceptionPolicyId { get; set; }
+
+        /// <summary>
+        /// A dictionary collection of exception rules on the exception policy. Key is the Id of each exception rule.
+        /// </summary>
+#pragma warning disable CA2227 // Collection properties should be read only
+        public IDictionary<string, ExceptionRule> ExceptionRules { get; set; } =
+#pragma warning restore CA2227 // Collection properties should be read only
```

Comment:
Based on our conversation, this will be brought up during SDK review to obtain feedback from arch board regarding user experience and will take action accordingly.